docs/nuget-org/nuget-org-faq.yml

```diff
@@ -263,3 +263,29 @@ sections:
           2. Click on this url: [https://www.nuget.org/account/delete](https://www.nuget.org/account/delete) and follow the steps to submit the request for deleting the account.
           
           Our customer support will process this request and perform the account deletion.
+          
+      - question: Error connecting to nuget.org or restoring packages from `https://api.nuget.org/v3/index.json`?
+        answer: |
+          If you are experiencing connectivity issues while connecting to nuget.org or restoring packages from https://api.nuget.org/v3/index.json, such as:
+
+          > "The SSL connection could not be established"
+
+          , or
+
+          > “[nuget.org] Unable to load the service index for source https://api.nuget.org/v3/index.ison. An error occurred while sending the request.
+The underlying connection was closed: An unexpected error occurred on a send. Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.
+An existing connection was forcibly closed by the remote host.”
+
+          , or
+
+          > “This page can’t be displayed. Turn on TLS 1.0, TLS 1.1, and TLS 1.2 in Advanced settings and try connecting to https://api.nuget.org again. If this error persists, it is possible that this site uses an unsupported protocol or cipher suite such as RC4 (link for the details), which is not considered secure. Please contact your site administrator.”
+
+        , this could be related to [the changes to nuget.org supported cipher suites](https://github.com/NuGet/Announcements/issues/60). 
+
+        Please ensure that TLS 1.2 and any of the following cipher suites have been enabled on your machine:
```

Comment:
Is it worth linking to the Azure DevOps TLS checker? https://github.com/microsoft/azure-devops-tls12

I don't know if it checks the same cyphers as what you listed below, but if it does, it might be an easy way for customers to check.  Despite all my knowledge of Windows, I honestly wouldn't have a clue how to check, and this docs update doesn't link to anything that tells customers how to check, though I recognise that it depends on the operating system, and maybe even which version of an operating system family.

Reply:
does https://www.ssllabs.com/ssltest/ cover this?

Reply:
No. The website tests your browser's TLS/SSL settings, but browsers don't use Windows' SChannel APIs, they ship their own SSL library (probably OpenSLL). So, checking that website in your browser does not validate that .NET apps can use the appropriate TLS ciphers.

Reply:
Ah. I just mostly meant for an end user to test what cipher suites have been enabled:

i.e.

![image](https://user-images.githubusercontent.com/1288848/193666924-8e3c4c74-21dd-422e-b3af-89639cfc5b4d.png)

Either way sounds good to me!

Reply:
> Ah. I just mostly meant for an end user to test what cipher suites have been enabled:

I'm not 100% sure about Edge (the new "Edgium", not the old Edge that is now deprecated), but I'm reasonably confident that Chrome and Firefox ship their own OpenSSL and therefore don't use Windows' configuration. So my point stands that opening that web page in a browser will test the browser's cipher suite, not the "Windows" configuration that .NET uses.

Reply:
Chrome has its own cipher suites, while .NET/Windows keeps a different set:
https://support.venafi.com/hc/en-us/articles/360046174112-Validate-Cipher-Suites-Offered-to-Servers-from-Windows